ansible/templates/cluster-template.yaml.j2

```diff
@@ -630,6 +630,15 @@ worker:
           RateLimitInterval=0
           RateLimitBurst=0
 
+#  Tuned the net.ipv4.tcp_retries2 kernel param on the worker nodes, so that stale TCP connections will be closed in ~ 1min.
+#  Since the kubelet holds a long TCP connection to the API server for posting its status, if that connection gets stalled, we'll get a NodeNotReady
+#  The default was 15 which caused the stale connections to be closed in ~ 15 mins.
+#  More on this here:  https://github.com/kubernetes/kubernetes/pull/48670, https://github.com/kubernetes/kubernetes/issues/41916#issuecomment-312428731
+      - path: /etc/sysctl.d/90-tcp-retries.conf
+        permissions: 0644
+        content: |
+          net.ipv4.tcp_retries2=6
```

Comment:
Does this mean we will need to wait up to 6 minutes before a node's connection is dropped and it reconnects?

Do we need to drop the retry count down lower?

Reply:
Not really, so a value of closes the connection in less than 60s as I tested. I wrote it in the comment :)

Reply:
Cool. 👍